### DISCUSSION TITLE
No Longer Replacing All Faces

### DISCUSSION THREAD

**Discussion by dougvan1701** [glucauze/sd-webui-faceswaplab#91] [Q&A]
I updated to the latest version of A1111 for SDXL and now FSL only swaps maybe 1 or two faces on an eight batch process. I have tried to increase the face number to see if it's somehow counting them wrong and disabled "Gender" in case this was incorrectly labeling them. Nothing works. I have a bare bones version of A111 running (base git pull with updates and only FSL) and FSL installed without errors.
Anyone else seeing this?
Thanks!

**Comment by glucauze** [comment]
This might be related to NSFW filtering, see : https://github.com/glucauze/sd-webui-faceswaplab/issues/80

**Reply by dougvan1701** [reply]
Thank you for the response!
I tried disabling it by setting it to 1 when I saw that thread last night while I was searching. It didn’t correct the issue. 
I didn’t however test the seed as was suggested in the thread. I will test that when I get a chance later today.

**Reply by dougvan1701** [reply]
I started to test the see but I noticed a gradio error flooding the console so instead of fighting with another issue I did another fresh install, set the NSFW filter to 1 and restarted the console. I have tested it several times and all faces are now swapped properly.
I am going to have to point the finger at something wrong with the installs or another extension being the culprit. Sorry to bother you with this but thank you for the help!
